src/NUnitFramework/nunit.framework.legacy/nunit.framework.legacy.csproj

```
@@ -6,14 +6,16 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
 
-  <ItemGroup>
-    <Compile Include="..\..\CommonAssemblyInfo.cs" Link="Properties\CommonAssemblyInfo.cs" />
-    <Compile Include="..\FrameworkVersion.cs" Link="Properties\FrameworkVersion.cs" />
-  </ItemGroup>
-
   <ItemGroup>
     <ProjectReference Include="..\framework\nunit.framework.csproj" />
   </ItemGroup>
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net462'">
+    <Title>NUnit Framework Classic (.NET Framework 4.6.2)</Title>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net6.0'">
+    <Title>NUnit Framework Classic (.NET 6.0)</Title>
+  </PropertyGroup>
```

Maybe we can use the previously set `AssemblyConfiguration` in order not to duplicate things:

```suggestion
  <PropertyGroup>
    <Title>NUnit Framework Classic $(AssemblyConfiguration)</Title>
  </PropertyGroup>
```